README.md

```diff
@@ -200,6 +201,12 @@ For example, a second release in June 2021 would be 202106.01. Although the SDK
 
 All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.1.2](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.2).
 
+### 202113.00
+
+[API documentation of 202113.00 release](https://docs.aws.amazon.com/embedded-csdk/202113.00/lib-ref/index.html)
+
+This release includes a [major update](https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/main/CHANGELOG.md#v300-march-2021) to the APIs of the AWS IoT Over-the-air Update library. Additionally, AWS IoT Device Shadow library introduces a [minor update](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/blob/main/CHANGELOG.md#v110-march-2021) by adding support for named shadow, a feature of the AWS IoT Device Shadow service that allows you to create multiple shadows for a single IoT device. AWS IoT Jobs library introduces a [minor update](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/blob/main/CHANGELOG.md#v110-march-2021) by introducing macros for `$next` job ID and compile-time generation of topic strings. AWS IoT Device Defender library introduces a [minor update](https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk/blob/main/CHANGELOG.md) that adds macros to API for custom metrics feature of AWS IoT Device Defender service. corePKCS11 also introduces a [major update](https://github.com/FreeRTOS/corePKCS11/blob/main/CHANGELOG.md#v301-february-2021) by removing the `pkcs11configPAL_DESTROY_SUPPORTED` config and mbedTLS platform abstraction layer of `DestroyObject`. Lastly, no code changes are introduced for backoffAlgorithm, coreHTTP, coreMQTT, and coreJSON but patch updates are made to improve documentation and CI.
```

Comment:
>This release includes a [major update](https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/main/CHANGELOG.md#v300-march-2021) to the APIs of the AWS IoT Over-the-air Update library.

It can be helpful to point to the CHANGELOG of OTA to point to the API changes?